src/devices/cpu/gigatron/gigatron.cpp

```diff
@@ -11,7 +11,7 @@
 #include "gigatrondasm.h"
 
 
-DEFINE_DEVICE_TYPE(GTRON, gigatron_cpu_device, "gigatron", "GTRON")
+DEFINE_DEVICE_TYPE(GTRON, gigatron_cpu_device, "gigatron", "Gigatron CPU Device")
```

Comment:
Currently there is a machine shortname conflict here.  "gigatron" cannot be both the actual machine name and a device name as devices are handled as machines internally and in XML output.  Please rename it to something like "gigatron_cpu" so the duplication can be fixed.

Reply:
Fixed.